job_templates/service_action_-_ssh_default.erb

```diff
@@ -19,5 +19,17 @@ template_inputs:
 <% if @host.operatingsystem.family == "Redhat" && @host.operatingsystem.major.to_i > 6 %>
 systemctl <%= input("action") %> <%= input("service") %>
 <% else %>
-service <%= input("service") %> <%= input("action") %>
+  <% case input("action")
+  when 'enable'
+%>
+chkconfig --add <%= input("service") %>
+  <%
+    when 'disable'
+  %>
+chkconfig --del <%= input("service") %>
+  <%
+      else
```

Comment:
The indentation in this template seems off

Reply:
I hope that it is better now, i did "auto format" before, and now by hand :)